apps/files/src/views/TemplatePicker.vue

```diff
@@ -243,7 +243,13 @@ export default defineComponent({
 
 				// Update files list
 				emit('files:node:created', node)
-				emit('files:node:focus', node)
+
+				// Open the new file
+				window.OCP.Files.Router.goToRoute(
+					null, // use default route
+					{ view: 'files', fileid: node.fileid },
+					{ dir: node.dirname, openfile: 'true' },
```

Comment:
```suggestion
					{ dir: node.dirname, openfile: true },
```
From the code it should be boolean? https://github.com/nextcloud/server/blob/d65e46f31367e65b03d4ab829baeacbece04850a/apps/files/src/components/FilesListVirtual.vue#L212

Reply:
the API doesn't accept boolean :)
But that did trigger an interesting question, and after checking, if you want `&openfile` and not `&openfile=true`, which I thought the boolean would do, you need to use `null` :see_no_evil: 

![image](https://github.com/nextcloud/server/assets/14975046/f5711ea2-50c5-4dd1-9a08-d584d345a1e6)

Reply:
Also, the openfile watcher is watching the computed, which doesn't care about the value, it just checks if it's there or not ;)
https://github.com/nextcloud/server/blob/8be4704e1125b6eb6fe17a58c8b54c651fa40c0c/apps/files/src/components/FilesListVirtual.vue#L159-L161